### PR TITLE
Create simple list with RecylerView and ViewBinding

### PR DESCRIPTION
**Pull request by YasmminClaudino**

### CODE REVIEW

**Review by ehammo — COMMENTED** [review]
We could change the commit message to Create simple list with RecyclerView and ViewBinding

What do you think?

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -6,13 +6,33 @@
...
         app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+        android:id="@+id/info"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
```
Comment (ehammo) [inline app/src/main/res/layout/activity_main.xml]: These TextViews are not being used. We could remove them right?

File: app/src/main/res/layout/rv_item_repo.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginTop="37dp"
+        android:text="TextView"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.045"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/title"
+        app:layout_constraintVertical_bias="1.0" />
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (ehammo) [inline app/src/main/res/layout/rv_item_repo.xml:117]: We could wait until you made a rv_item_repo a little similar to the mockup right? What do you think?

Comment (YasmminClaudino) [reply app/src/main/res/layout/rv_item_repo.xml:117]: I added new items to the list however I had a little bit of difficult to add the other 3 items with the constraint Layout: Username, full name and user photo. 
Will try a new commit tomorrow to add the missing items.

**Pull request closed without merging**